platforms/documentation/docs/src/snippets/how-to/share-outputs/groovy/consumer/build.gradle

```
@@ -3,26 +3,34 @@ plugins {
     id("application")
 }
 
-configurations {
-    create("instrumentedRuntime") {
-        canBeConsumed = false
-        canBeResolved = true
-        attributes {
-            attribute(Usage.USAGE_ATTRIBUTE, objects.named(Usage, Usage.JAVA_RUNTIME))
-            attribute(LibraryElements.LIBRARY_ELEMENTS_ATTRIBUTE, objects.named(LibraryElements, "instrumented-jar"))
-        }
+// This configuration is used to declare dependencies only.
+def instrumentedRuntimeDependencies = configurations.create("instrumentedRuntimeDependencies") {
```

```suggestion
def instrumentedRuntimeDependencies = configurations.dependencyScope("instrumentedRuntimeDependencies")
```

Then no need to set flags manually.